fdbserver/ptxn/test/TestTLogServer.actor.cpp

```diff
@@ -789,29 +792,51 @@ TEST_CASE("/fdbserver/ptxn/test/pop_data") {
 	state ptxn::test::TestDriverOptions options(params);
 	state std::vector<Future<Void>> actors;
 	(const_cast<ServerKnobs*> SERVER_KNOBS)->BUGGIFY_TLOG_STORAGE_MIN_UPDATE_INTERVAL = 0.5;
+	(const_cast<ServerKnobs*> SERVER_KNOBS)->TLOG_SPILL_THRESHOLD = 1500e6; // disable spilling
```

Comment:
Do you need to put this here? Isn't this the default value?

Reply:
yeah, randomness would change the knob to 0 and enable spilling inadvertently